data/input_files/tests/nonlinear_solid/dyn_solve.lua

```diff
@@ -82,9 +84,11 @@ nonlinear_solid = {
         ['displacement'] = {
             -- boundary attribute 1 (index 0) is fixed (Dirichlet) in the x direction
             attrs = {1},
-            vec_coef = function (v)
-                return Vector.new(0, 0, 0)
-            end
+            constant_vector = {
```

Comment:
Is "constant_vector" descriptive enough?

Reply:
I think so. This block describes coefficients, where `constant_vector` is a way to describe one. However, I'm open to other ideas if you have them!